.github/CODEOWNERS

```diff
@@ -395,6 +395,36 @@ flang/**/CodeGen/ @llvm/pr-subscribers-flang-codegen
 /llvm/utils/TableGen/CodeGenDAG* @llvm/pr-subscribers-selectiondag
 /llvm/utils/TableGen/DAGISel* @llvm/pr-subscribers-selectiondag
 
+# register allocation
+/llvm/include/CodeGen/RegAlloc* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/*Regalloc* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/Register* @llvm/pr-subscribers-regalloc
+/llvm/include/CodeGen/LiveReg* @llvm/pr-subscribers-regalloc
```

Comment:
How about `Live*`?

Reply:
That pulls in some unrelated debug info stuff